python/samples/concepts/memory/new_memory.py

```diff
@@ -70,9 +71,9 @@ class MyDataModelList:
 # configuration,
 # specify which store (redis_json, redis_hash, qdrant, Azure AI Search) to use
```

Comment:
should the comment be updated to include `volatile` as an option? I know it's specified below, but for completeness.

Reply:
good point, fixed